### PR TITLE
[SD-1443] Move Spree Decorators in Spree Gateway to SpreeGateway namespace

### PR DESCRIPTION
**Pull request by pmilewski**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,9 @@
 source 'https://rubygems.org'
 
 gem 'rails-controller-testing'
-gem 'spree', github: 'spree/spree', branch: 'master'
+gem 'spree', github: 'spree/spree', branch: 'main'
+gem 'spree_backend', github: 'spree/spree', branch: 'main'
+gem 'spree_frontend', github: 'spree/spree', branch: 'main'
+gem 'spree_emails', github: 'spree/spree', branch: 'main'
```
Comment (damianlegawiec) [inline Gemfile:7]: I think we don't need emails here

File: vendor/assets/javascripts/spree/backend/all.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// This is a manifest file that'll be compiled into including all the files listed below.
```
Comment (damianlegawiec) [inline vendor/assets/javascripts/spree/backend/all.js:1]: These files aren't needed they are automatically added by generators

**Review by damianlegawiec — CHANGES_REQUESTED** [review]

File: vendor/assets/javascripts/spree/backend/all.js (hunk 2 above)
Comment (pmilewski) [reply vendor/assets/javascripts/spree/backend/all.js]: For some reason these files are not added by generators and [specs failed](https://app.travis-ci.com/github/spree/spree_gateway/jobs/534731005#L5715
).
Should I do any change in the configuration?

**Comment by damianlegawiec** [comment]
@pmilewski https://github.com/spree/spree_auth_devise/commit/afed80ae8be4617986c77849a2d977054bae0ffd

**Review by damianlegawiec — APPROVED** [review]

**Pull request merged**